server/Cargo.toml

```diff
@@ -36,6 +36,9 @@ large_enum_variant = "allow"
 module_inception = "allow"
 bool_assert_comparison = "allow"
 result_large_err = "allow"
+ptr_arg = "allow"
+# TODO: Will need frontend changes too. Will be nice for readability.
+too_many_arguments = "allow"
```

Comment:
ptr_arg: sounds like a nice lint error, but I guess too many places where we violate it? maybe another refactor? but then we don't usually reuse our functions so its not really needed and would be a bit of work...

too_many_arguments: wonder if we should disable this case by case? Can't be that many places where we have too many arguments?

Reply:
Yup, I think ptr_arg would be nice to have in eventually too~. Though it tells you to use slice as the variable to pass in then in the method has you convert that to_vec 🤔 

I think it's fine to have it globally for now...